application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -46,23 +63,11 @@ public function __construct(AdminController $controller)
      */
     public function get_session_key($username, $password, $plugin = 'Authdb')
     {
-        $username = (string) $username;
-        $password = (string) $password;
-        $loginResult = $this->_doLogin($username, $password, $plugin);
-        if ($loginResult === true) {
-            $this->_jumpStartSession($username);
-            $sSessionKey = Yii::app()->securityManager->generateRandomString(32);
-            $session = new Session();
-            $session->id = $sSessionKey;
-            $session->expire = time() + (int) Yii::app()->getConfig('iSessionExpirationTime', ini_get('session.gc_maxlifetime'));
-            $session->data = $username;
-            $session->save();
-            return $sSessionKey;
-        }
-        if (is_string($loginResult)) {
-            return array('status' => $loginResult);
-        }
-        return array('status' => 'Invalid user name or password');
+        return (new SessionKeyCreate)
```

Comment:
Hard-coding command name here is totally fine, I think. We need to keep the old functions anyway.